src/Ormolu/Diff/ParseResult.hs

```diff
@@ -30,7 +27,7 @@ data ParseResultDiff
   = -- | Two parse results are the same
     Same
   | -- | Two parse results differ
-    Different [SrcSpan]
+    Different [RealSrcSpan]
```

Comment:
This seems to be a bit more precise typing because `isGoodSpan` selects `RealSrcSpan`s anyway. Afterwards we can check things like line numbers using total functions.